### ISSUE TITLE
The focus is shifted back to the designer tab after performing delete action on choice items in property-grid tab

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#832]
See the https://surveyjs.answerdesk.io/ticket/details/t4342/accessibility-issue-survey-designer-unable-to-delete-options-in-property-grid-tab thread for more details
